styles/themes/theme.tsx

```diff
@@ -13,6 +13,12 @@ declare module '@mui/material/styles' {
   interface PaletteOptions {
     link?: PaletteOptions['primary'];
   }
+  interface Theme {
```

Comment:
Can you add some spacing between the above and below.

Reply:
done